README.md

```diff
@@ -29,7 +29,7 @@ docs locally.
 
```

Comment:
When it comes to the `history/_index.md` file, I think it's good idea to also do the `infracloudio`->`kubeshop` renaming. In all cases (except one, when `Infracloud is in the GH username) it is about the GH issue/PR location after all. Please also change it 👍